core/scripts/chaincli/handler/keeper.go

```diff
@@ -171,27 +171,23 @@ func (k *Keeper) approveFunds(ctx context.Context, registryAddr common.Address)
 
 // deployRegistry20 deploys a version 2.0 keeper registry
 func (k *Keeper) deployRegistry20(ctx context.Context) (common.Address, *registry20.KeeperRegistry) {
-	registryLogic := k.cfg.KeeperRegistryLogicAddr
```

Comment:
registry logic is an internal detail. In CLI, user should either give the end registry address, or deploy a new registry (including logic)